pylint/checkers/strings.py

```diff
@@ -204,6 +205,12 @@
         "Used when we detect an f-string that does not use any interpolation variables, "
         "in which case it can be either a normal string or a bug in the code.",
     ),
+    "W1310": (
+        "Using an string which should probably be a f-string",
```

Comment:
```suggestion
        "The %s syntax imply an f-string but the leading 'f' is missing",
```
If we could provide the "{local_var}" as an argument it would be great so the message is easier to understand.

Ie final message would be:
``The '{var}' and '{var}' syntax imply an f-string but the leading 'f' is missing``
``The '{var}' and '{var_two}' syntax imply an f-string but the leading 'f' is missing``